test/e2e/azure_test.go

```diff
@@ -193,7 +193,7 @@ var _ = Describe("Workload cluster creation", func() {
 		fmt.Fprintf(GinkgoWriter, "INFO: skipping test requires pushing container images to external repository")
 	}
 
-	It("With 3 control-plane nodes and 2 Linux and 2 Windows worker nodes", func() {
+	It("With 3 control-plane nodes and 2 Linux and 2 Windows worker nodes [REQUIRED]", func() {
```

Comment:
Why is this one at `It` level and others are `Context`?

Reply:
Ah looking further it looks like this isn't part of this PR but was created in a different way from the other tests.  minor but might be worth following the same pattern

Reply:
Right, I'm just injecting these additional substrings at the place where the current test strings are that we use for regex matching.

Reply:
Could we add a parent Context here and inject the suffix there instead?

```
Context("Creating a highly available cluster [REQUIRED]", func() {
```

Reply:
Yes, done